src/pages/student/Dashboard.scss

```
@@ -44,6 +44,12 @@
     color: #8C8C8C;
 }
 
+.dashboard-blobblue-img {
+    width: 18vw;
+    transform: scaleX(-1) rotate(90deg);
+    height: 42vw;
+}
+
```

The sizing of images looks super weird when the page gets zoomed out, so let's replace the vw stuff with fixed pixels instead.

```suggestion
.dashboard-blobblue-img {
    position: absolute;
    z-index: -1;
    height: 200px;
    width: 500px;
}
```